### PR TITLE
doc: Add CODE_OF_CONDUCT to project repo

### PR DESCRIPTION
**Pull request by dbkinder**
Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

File: CODE_OF_CONDUCT.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
+Contributor Covenant Code of Conduct
+####################################
+
+Our Pledge
+**********
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
```
Comment (fitchbe) [inline CODE_OF_CONDUCT.rst:8]: making --> make (One _commits_ to making; one _pledges_ to  make.)

File: CODE_OF_CONDUCT.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Attribution
+***********
+
+This Code of Conduct is adapted from the `Contributor Covenant <homepage>`_, version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+
+.. _homepage: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see
+https://www.contributor-covenant.org/faq
```
Comment (fitchbe) [inline CODE_OF_CONDUCT.rst:85]: Missing period at the end.

**Review by fitchbe — APPROVED** [review]

File: CODE_OF_CONDUCT.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+Attribution
+***********
+
+This Code of Conduct is adapted from the `Contributor Covenant <homepage>`_, version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
```
Comment (gvancuts) [inline CODE_OF_CONDUCT.rst:80]: Do we not need a period at the end of this sentence?

**Review by gvancuts — APPROVED** [review]
LGTM, thanks!

File: CODE_OF_CONDUCT.rst (hunk 3 above)
Comment (fitchbe) [reply CODE_OF_CONDUCT.rst:80]: Yep; good catch.

**Comment by dbkinder** [comment]
@gvancuts @fitchbe thanks for the review, comments addressed.

**Pull request merged**